performance-timeline/case-sensitivity.any.js

```diff
@@ -27,3 +27,74 @@
     assert_equals(mixedList.length, 0, "getEntriesByName('" + location2 + "').length");
 
   }, "getEntriesByName values are case sensitive");
+
+  async_test(function (t) {
+    let observerCalled = false;
+    const observer = new PerformanceObserver(
+      t.step_func(function (entryList, obs) {
+        observerCalled = true;
```

Comment:
nit: perhaps this can use assert_unreached? Then no need for `observerCalled`.

Reply:
Nice, done.